server/opts_test.go

```diff
@@ -3256,3 +3256,31 @@ func TestMaxSubTokens(t *testing.T) {
 		t.Fatal("Did not get the permissions error")
 	}
 }
+
+func TestStringStoreSize(t *testing.T) {
+	tt := []struct {
+		input string
+		want  int64
+		err   bool
+	}{
+		{"1K", 1024, false},
+		{"1M", 1048576, false},
+		{"1G", 1073741824, false},
+		{"1T", 1099511627776, false},
+		{"1L", 0, true},
```

Comment:
May want to add `{"", 0, false}` to the list?

Reply:
added.